solidity/random-beacon/test/data/bls.ts

```diff
@@ -14,13 +26,43 @@ const blsData = {
   groupSignature:
     "0x112d462728e89432b0fe40251eeb6608aed4560f3dc833a9877f5010ace9b1312006dbbe2f30c6e0e3e7ec47dc078b7b6b773379d44d64e44ec4e017bfa7375c",
 
+  // group signature over groupSignature
+  nextGroupSignature:
+    "0x144b0508140c2c63fac298ee0cdd98571971a2d958f7c97d2bab82f3e1e727542d08314d6f087aca6ec2173b9a1d928cb80ff45258984a8929977a58d8b2fc26",
+
   // group signature over previousEntry uint256
   groupSignatureUint256:
     "33136845259729814081977577432759716433762925710284529101628804651946005705295",
 
-  // group signature over groupSignature
-  nextGroupSignature:
+  groupSignatures: [
+    // signed by group with 'groupPubKey'
+    "0x112d462728e89432b0fe40251eeb6608aed4560f3dc833a9877f5010ace9b1312006dbbe2f30c6e0e3e7ec47dc078b7b6b773379d44d64e44ec4e017bfa7375c",
     "0x144b0508140c2c63fac298ee0cdd98571971a2d958f7c97d2bab82f3e1e727542d08314d6f087aca6ec2173b9a1d928cb80ff45258984a8929977a58d8b2fc26",
+    "0x10bbc10ee3e5509fffa43f797a5967c46f94b19a01d40360f88d0b13a2a5dc491112a6540550709ae6673af3d4e90ef96c7d7593b71ba6e335276d0b5fc5f3ae",
+    "0x2ed17b6237a4b9b9389f6964c4a07017d8461cb2602c367fbce7f9414585f6fc0ba88e1264f783386fb8493d31d733a100bf21f46acf0b3bf99c89834517907f",
+    "0x16a2fce00c71da31748aa890186fedaddb4960ed6308cb4826cbe941147979f52f52d1ae746fa75a25bfa30515ffadd4c58db9a45adb3d8cbe779263a273ba3a",
+    "0x1954d2a22636ebfba4bb3c44ca71863729aa8f638080d67a98ee267cf5b6df0d1c711f2308bab428ee822871bb22efa8121d1e58c198e1745843c5b37efb67fc",
+    // signed by group with 'groupPubKey2'
+    "0x119dc6318071d71cfe8deabbb2264b84041f5bc52eeab84c6a01932163ea874a23013e859d5906dd183e36cab2ccf3d2a580429aa6a1ae25fb76b485e2f7520d",
+    // signed by group with 'groupPubKey2'
+    "0x08875f9e43e3498c4a139cc76e5311c530bbb589236213b7b668ce216b7a07902d280aaad20d2b091daa8dcc729d383e9da57482f07c32aee27acd8d71f45ba2",
+    // signed by group with 'groupPubKey2'
+    "0x15bbb764a647b078a37e96adc9f5eda5863d3b1fd0d027eabc15010da4f35afb07cb7e1660d5776f1ca91bbd255f79dc669a8197ce8589230f94f7a31f213029",
+    // signed by group with 'groupPubKey'
+    "0x0c62b62095142271bb6ed0474e2ce6299395a83d233c1b12da41d49e8516cfa80c27ff0ce575d5c686b0be9e68f8328a92056b8093c16741b8756df7a066c196",
+    // signed by group with 'groupPubKey3'
+    "0x25b12af46496104317227c716c85a971c2a74908ad43c2ca6d9ac830a6a3650f0ce79fb5dd2c18eeecbcbf5f586dea29aa09e69e97cc14e7f0e670fb8e7bc189",
+    // signed by group with 'groupPubKey1'
+    "0x245638c2a457cad293e140749d2ac45e851d73cc259f4202ac497238b68655c82ad7835b7431bf90d97890199d5354939494b8570b771013671bc9161e0c2406",
```

Comment:
Are those `// signed by group with ...` correct?

Reply:
yes, I reworded a bit, but the general idea is that every 5th relay request a new group is added. The following equation `groupId = prevEntry % numberOfActiveGroups` determines which active group signs the prev entry. Group pub key is sort of a unique id and that's why I added comments to denote which group signed a prev relay.